istio/istio-daemonset.yml

```diff
@@ -0,0 +1,117 @@
+# runs linkerd in a daemonset, in linker-to-linker mode
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+data:
+  config.yaml: |-
```

Comment:
I'm a little concerned about doing this, since in the optimal case users of istio-linkerd will never even look at the yaml... so they won't have the option to change it. These configs feel less informational and more utilitarian (You could argue we should put them somewhere besides linkerd-examples).